### ISSUE TITLE
Can not install Safari Driver extension (2.48) in Safari 9.1.1 (OS X Yosemite 10.10.5)

### ISSUE THREAD

**Issue by abhishekswain** [SeleniumHQ/selenium#2174]
## Meta - browser-safari
OS:  Mac OS X Yosemite 10.10.5
Selenium Version:  Selenium Server 2.47.1
Browser:  Safari
Browser Version:  9.1.1 (64 bit)

## Expected Behavior -
Safari Driver extension should get installed.

## Actual Behavior -
Extension is not getting installed.

## Steps to reproduce -
1. Download Safari driver extension.
2. Double click to install
3. Click on the trust button

After clicking on trust button nothing happens. Extension does not get installed.
Seems it is a Safari issue after “ Security Update 2016-003"

**Comment by 5hawnknight** [comment]
I have the same OS/Browser/Security Update configuration and I am able to installed the extension without issue. Was able to uninstall and reinstall the extension several times, again without issue.

I used the following download site -- http://selenium-release.storage.googleapis.com/index.html?path=2.48/

**Comment by frobibar** [comment]
Hi, I'm getting the same behaviour "After clicking on trust button nothing happens.".

OS X: El Capitan 10.11.4 (15E65)
Safari: Version 9.1 (11601.5.17.1)
SafariDriver.safariextz: Version 2.48

**Comment by frobibar** [comment]
Hi, I found the cause of the problem (for me). Its related to using Screen Sharing to install the extension. A small additional catch, it also fails to install even when you have a Screen Sharing session running but not actually using it to install the extension.

So don't use Screen Sharing (and don't have it running) and connect directly solved my issue. One other option if direct access is a problem, is to setup VNC and try to install it via VNC to see if might avoid the problem (I have not tried this myself).

BTW, I was able to use Screen Sharing to install on a machine with the following:
OS X: Yosemite 10.10.5
Safari: Version 8.0.8 (10600.8.9)
SafariDriver.safariextz: Version 2.48
